pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -329,6 +360,30 @@ func newRunnerPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID, aEx
 				WithArgumentsNew([]string{"-c", "ansible-playbook ./executor/test.yml -i /etc/ansible/hosts; exit 0"}).
 				WithEnvsNew(getChaosRunnerENV(cr, aExList)),
 		).
+		Build()
+	if err != nil {
+		return nil, err
+	}
+	return runnerPod, nil
+}
+func newExporterPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID) (*corev1.Pod, error) {
```

Comment:
Should we consider making this a deployment? Pod is good enough in terms of reconcile, but lacks ability to specify selector..so this may still not be desirable for older k8s versions. The initial idea was to move to a job (which supports selector) which will cause the sidecar to remain alive - however, this is a more elegant solution to separate out the monitoring piece to a separate resource.

Reply:
Will chage that, Please check the new PR